.gitignore

```diff
@@ -16,3 +16,7 @@ coverageEnv
 .idea
 .DS_Store
 /scripts/.ropsten-test.env
+
+# Generated by Certora
+.certora_*
+.last_confs
```

Comment:
Certora doesn't seem to provide a way to create those files elsewhere out-of-the-box, so they will be ignored for now